website_product_share/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Product Share
+============
+Allow users to enable/disable share button on each product page
+
+Usage
+=====
+
+* Go to website
+* Open one of the product
+* Click on customize button
+* Toggle share product option to enable/disable share button
+
```

Comment:
NB: even if you have to go to one specific product, the set up is valid for all of them.